doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,148 @@
+[id='test-scenarios-comparison-legacy-new-ref']
+
+= Feature comparison of legacy and new test scenario designer
+
+{PRODUCT} currently supports both the new *Test Scenarios* designer and the former *Test Scenarios (Legacy)* designer.
+
+The default designer is the new test scenarios designer, which supports testing of both rules and DMN models and provides an enhanced overall user experience with test scenarios. If required, you can continue to use the legacy test scenarios designer, which supports rule-based test scenarios only.
+
+IMPORTANT: The legacy Test Scenarios tool was deprecated with {PRODUCT} 7.3.0. It will be removed in a future {PRODUCT} release. Use the new Test Scenarios designer instead.
```

Comment:
Suggested rewrite:

The new test scenarios designer has an improved layout and feature set and continues to be developed. However, the legacy Test Scenarios designer is deprecated with Red Hat Process Automation Manager 7.3.0. and will be removed in a future Red Hat Process Automation Manager release.

Reply:
Done.